bin/webpack-configs.js

```diff
@@ -367,7 +365,39 @@ const getFrontConfig = ( options = {} ) => {
 			concatenateModules:
 				isProduction && ! process.env.WP_BUNDLE_ANALYZER,
 			splitChunks: {
-				automaticNameDelimiter: '--',
+				cacheGroups: {
+					default: false,
+					vendors: {
+						test: /[\\/]node_modules[\\/]/,
+						name( module ) {
+							const moduleFileName = module
+								.identifier()
+								.split( '/' )
+								.reduceRight( ( item ) => item );
+							return `vendor/${ moduleFileName }`;
+						},
+						automaticNameDelimiter: '--',
+						minSize: 20000,
+						priority: -20,
+					},
+					'cart-checkout-wp-commons': {
+						test: ( module ) => {
+							if (
+								module?.resource?.match(
+									/wordpress-components/
+								)
+							) {
+								return true;
+							}
+						},
+						name: 'cart-checkout/commons.js',
+						automaticNameDelimiter: '--',
+						reuseExistingChunk: true,
+						enforce: true,
+						minSize: 20000,
+						priority: -10,
+					},
+				},
```

Comment:
This groups all wordpress-components into a single file, I should try to add emotion to this as well.